### PR TITLE
Sklearn normalize depreciation fix

### PR DESCRIPTION
**Pull request by namanmistry**
### Description

The new version of sklearn's LinearRegression does not allow `normalize` paramter which is depreciated in the newer versions. So I fixed that.

### Related issues or pull requests

`Fixes Arg normalize to sklearn.linear_model.LinearRegression has been removed in version 1.2 #1035`

### Pull Request Checklist

- [x] Added a note about the modification or contribution to the `./docs/sources/CHANGELOG.md` file (if applicable)
- [x] Added appropriate unit test functions in the `./mlxtend/*/tests` directories (if applicable)
- [x] Modify documentation in the corresponding Jupyter Notebook under `mlxtend/docs/sources/` (if applicable)
- [x] Ran `PYTHONPATH='.' pytest ./mlxtend -sv` and make sure that all unit tests pass (for small modifications, it might be sufficient to only run the specific test file, e.g., `PYTHONPATH='.' pytest ./mlxtend/classifier/tests/test_stacking_cv_classifier.py -sv`)
- [x] Checked for style issues by running `flake8 ./mlxtend`

### CODE REVIEW

**Comment by rasbt** [comment]
Thanks for the PR, it looks good to me! Just moved the Changelog entry to the right section.

**Review by rasbt — APPROVED** [review]
